tests/core/test_actions.py

```diff
@@ -344,6 +362,156 @@ async def test_remote_action_with_template_param(
     ]
 
 
+@pytest.mark.parametrize("event_class", rasa.shared.utils.common.all_subclasses(Event))
+async def test_remote_action_validate_all_event_subclasses(event_class: Type[Event]):
```

Comment:
should we move this test to a module `tests.shared.utils.schemas.events`? The test doesn't seem particularly related to the action server response

Reply:
Ok, done 👍